WordPress/src/main/java/org/wordpress/android/ui/main/jetpack/migration/JetpackMigrationViewModel.kt

```diff
@@ -121,11 +129,26 @@ class JetpackMigrationViewModel @Inject constructor(
 
     @Suppress("ForbiddenComment")
     private fun onContinueFromNotificationsClicked() {
-        // TODO: Disable notifications in WP app
-        //  See https://github.com/wordpress-mobile/WordPress-Android/pull/17371
+        if (preventDuplicateNotifsFeatureConfig.isEnabled()) disableNotificationsOnWP()
         postDoneState()
     }
 
+    private fun disableNotificationsOnWP() {
+        AppLog.d(T.NOTIFS, "Disable Notifications")
+        Intent().also { intent ->
+            intent.action = "org.wordpress.android.broadcast.DISABLE_NOTIFICATIONS"
+            val appSuffix = BuildConfig.APPLICATION_ID.split(".").last()
+            val appPackage = if (appSuffix.isNotBlank()) {
+                "org.wordpress.android.${appSuffix}"
+            } else {
+                "org.wordpress.android"
+            }
```

Comment:
<img width="358" alt="image" src="https://user-images.githubusercontent.com/2471769/203554562-46033160-cef7-4137-a063-d3c4b8bcca86.png">
This block doesn't work for the vanilla debug. 

What do you think about this solution? ⬇️ 
```
BuildConfig.APPLICATION_ID.replace("com.jetpack", "org.wordpress")
```

Reply:
This is supposed to set receiving application Id, which is `org.wordpress.android` , this is meant to add .prealpha for jalapeno and .beta for wasabi variants only.  Vanilla beta and release would have just `org.wordpress.android`.  Unless I'm missing something here!